paper-toast.html

```diff
@@ -112,10 +111,13 @@
       Polymer.IronA11yAnnouncer.requestAvailability();
     },
 
-    ready: function() {
-      this.async(function() {
+    attached: function() {
+      var visible = (this.hasAttribute('visible') && this.getAttribute('visible') !== 'false');
```

Comment:
`this.visible` is read-only, hence it won't be set, so we have to check for the attribute here